tools/ci_build/builds/release_linux.sh

```diff
@@ -43,12 +43,7 @@ python3 -m pip install --upgrade setuptools
 python3 --version
 python3 ./configure.py
 
-## Run bazel test command. Double test timeouts to avoid flakes.
-bazel test -c opt -k \
-    --jobs=auto --test_timeout 300,450,1200,3600 \
-    --test_output=errors --local_test_jobs=8 \
-    --crosstool_top=//build_deps/toolchains/gcc7_manylinux2010-nvcc-cuda10.1:toolchain \
-    //tensorflow_addons/...
+bash tools/ci_testing/addons_cpu.sh
```

Comment:
I would think this is an issue. `addons_cpu` will compile the custom-ops without any particular toolchain:
https://github.com/tensorflow/addons/blob/master/tools/ci_testing/addons_cpu.sh#L60-L64

I thought the compile at the subsequent build since they're already created, but they won't be compatible with tf from pypi. However the test-release-wheel is passing, so I wanted to verify with you that this isn't reverting to python_op fallback?

Reply:
Could we add a `bazel clean --expunge` after this just for sanity.

Reply:
Sure thing. I'll make sure the right toolchaine is used.